packages/client/src/pages/sign_up/sign_up.tsx

```diff
@@ -0,0 +1,115 @@
+import Container from 'react-bootstrap/Container'
+import Form from 'react-bootstrap/Form'
+import Button from 'react-bootstrap/Button'
+import Row from 'react-bootstrap/Row'
+import Col from 'react-bootstrap/Col'
+import { useForm } from 'react-hook-form'
+import '../../../../../node_modules/bootstrap/dist/css/bootstrap.min.css'
```

Comment:
Такое лучше в index.ts/tsx подключать

  и сократить путь до такого
```suggestion
import 'bootstrap/dist/css/bootstrap.min.css';
```

Reply:
Исправил. Но это временный импорт. В продакшене будет линк на CDN.